### PR TITLE
Add docs about wagtailforms customisation

### LINKED ISSUE #2945: Documentation for new features of wagtailforms
We need to move documentation from [`wagtailsurveys`](https://github.com/torchbox/wagtailsurveys) (see `README.md`) into Wagtail once #2617 will be merged.

### PR DESCRIPTION
**Pull request by m1kola**
Fixes #2945

### CODE REVIEW

File: docs/reference/contrib/forms/customisation.rst [hunk 1]
```diff
@@ -0,0 +1,554 @@
+Form builder customisation
+==========================
+
+For basic usage example see :ref:`form_builder_usage`.
```
Comment (tomdyson) [inline docs/reference/contrib/forms/customisation.rst]: for basic -> for a basic

**Comment by m1kola** [comment]
Thanks @tomdyson and @timorieber for the review. I've updated the PR.

File: docs/reference/contrib/forms/customisation.rst [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+                FieldPanel('subject'),
+            ], "Email"),
+        ]
+
+        def get_form_class_for_step(self, step):
+            return self.form_builder(step.object_list).get_form_class()
+
+        def serve(self, request, *args, **kwargs):
+            """
+            Implements simple a multi-step form.
```
Comment (tomdyson) [inline docs/reference/contrib/forms/customisation.rst]: simple a -> a simple

**Comment by m1kola** [comment]
Merged in 6ee25ccab91d315fb4a3f794301471c60965880d + parents

Thanks @tomdyson and @timorieber for the review!

**Pull request closed without merging**